src/examples/TimeServer/TimeServer/TimeServer.csproj

```diff
@@ -43,8 +43,8 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    
```

Comment:
You should remove all Compile items only in the new projects. All examples still use the old one